src/argus_htmx/themes/utils.py

```diff
@@ -1,17 +1,42 @@
+from importlib.resources import files
+
 from django.conf import settings
-from argus_htmx import settings as argus_htmx_settings
+from django.core.exceptions import ImproperlyConfigured
 
+from argus_htmx import settings as default_htmx_settings
 
-def get_themes():
-    return getattr(settings, "DAISYUI_THEMES", argus_htmx_settings.DAISYUI_THEMES)
 
+__all__ = ["get_theme_names"]
 
-def get_theme_names():
-    themes = get_themes()
+
+def get_themes_from_setting():
+    themes_setting = getattr(settings, "DAISYUI_THEMES", default_htmx_settings.DAISYUI_THEMES)
     theme_names = []
-    for theme in themes:
+    for theme in themes_setting:
         if isinstance(theme, str):
             theme_names.append(theme)
         elif isinstance(theme, dict):
             theme_names.extend(theme.keys())
     return theme_names
+
+
+def get_themes_from_css():
+    styles_css = files("argus_htmx").joinpath("static/styles.css").read_text()
```

Comment:
The css file is overridable using the `path_to_stylesheet` context variable, so this should respect that

Reply:
Template context can never be available at this stage in the loading of the app.

We'd need to change how `path_to_stylesheet` is set, if not `path_to_stylesheet` is also only valid after `collectstatic` is run.

Reply:
I would be ok doing that through a setting

Reply:
Setting made, mentioned in readme.